src/renderer/components/+catalog/catalog.module.css

```diff
@@ -19,43 +19,59 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
-.list :global(.TableRow) {
```

Comment:
Instead of all these changes, wouldn't it just have been easier to use `showWithColumn`?

Reply:
No. Because otherwise it' requires other hacks to hide text "Icon" in the header column or if we leave it empty as result we have empty menu-item for the table.

Reply:
`showWithColumn` hides that menu item that is the point of it.